proposals/containerd.adoc

```diff
@@ -0,0 +1,55 @@
+== containerd Proposal
+
+*Name of project:* containerd
+
+*Description*
+
+containerd is an industry-standard container runtime with an emphasis on simplicity, robustness and portability. It is available as a daemon for Linux and Windows, which can manage the complete container lifecycle of its host system: image transfer and storage, container execution and supervision, and low-level storage, etc..
+
+containerd is designed to be embedded into a larger system, rather than being used directly by developers or end-users. containerd will serve as a core container runtime for the CNCF ecosystem.
+
+*Sponsor / Advisor from TOC*: Brian Grant
+
+*Unique Identifier*: containerd
+
+*License*: Apache License v2.0
+
+*Source control repositories*: https://github.com/docker/containerd
```

Comment:
Sorry I can't edit my comment for typos...

s/limix/linux/

s/Docker project/Docker github org/